packages/types/src/payment-types.ts

```diff
@@ -54,12 +32,60 @@ export interface IFeeReferenceBasedCreationParameters extends IReferenceBasedCre
 }
 
 /** Parameters to create a request with "any to erc20" payment network */
-export interface IAnyToErc20CreationParameters extends IFeeReferenceBasedCreationParameters {
-  network?: string;
+export interface IAnyToErc20CreationParameters extends ICreationParametersAnyToAny {
   acceptedTokens?: string[];
-  maxRateTimespan?: number;
 }
 
+/** Interface to create a payment network  */
+export interface IPaymentNetworkCreateParameters<T = any> {
+  id: PAYMENT_NETWORK_ID;
+  parameters: T;
+}
+
+export type PaymentNetworkCreateParameters =
```

Comment:
This is the main change. We strongly type creation parameters based on the different payment networks.